queries/csharp.scm

```diff
@@ -274,8 +274,10 @@
 
 (_
   (parameter_list
+    .
```

Comment:
I'm not sure if I prefer `.` in these cases. It doesn't really change the result and it is more verbose. On the other hand it is more explicit.